spec/models/many_to_many/reader.rb

```diff
@@ -0,0 +1,4 @@
+class Reader < ActiveRecord::Base
```

Comment:
I might be reading this wrong, but it looks like the `Reader` class isn't actually required for the test scenario here, is it?

Reply:
Actually, the problem with this scenario cannot be solved so easily, and the implementation is too naive. In the scenario, we have a `ReadersArticle` that updates the `readers_count` counter. Everything works fine until we delete the associated `AuthorsArticle`, which leads to the counter becoming out of sync.

Therefore, I don't see much benefit in this functionality. Unless leaving the technical possibility of specifying a multi-level path for the counter, and letting the implementation be up to a user, as it should be quite complex.

Reply:
Should we close this then?